README.md

```diff
@@ -75,9 +75,15 @@ A Ruby script that colorizes the `ls` output with color and icons. Here are the
     rehash
     ```
 
-4. Start using `colorls` :tada:
+4. Enable tab completion for flags by entering following line to your shell configuration file (`~/.bashrc`, `~/.zshrc` or `~/.fishrc`) :
```

Comment:
`~/.fishrc` is NOT the shell configuration file for fish. You can leave that out.